src/routes/(admin)/account/AccountForm.svelte

```
@@ -0,0 +1,56 @@
+<script lang="ts">
+	import { Button } from '$lib/components/ui/button';
+	import * as Form from '$lib/components/ui/form/index.js';
+	import { Input } from '$lib/components/ui/input/index.js';
+	import { type Infer, superForm } from 'sveltekit-superforms';
+	import { zodClient } from 'sveltekit-superforms/adapters';
+	import { editUserSchema, type EditUserSchema } from './schema';
+
+	let { data, isEditMode = $bindable() }: { data: any; isEditMode: boolean } = $props();
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)